### PR TITLE
created error page

### PR DESCRIPTION
**Pull request by shoutoutloud**
**What**: added 404 error page.

**Why**: to enhance user experience.

**How**: using nunjucks.

**Checklist**:

- [ ] Documentation
- [x] Ready to be merged
      
- [ ] Added myself to contributors table
      

Thank you for giving me a chance to contribute Please have a look.

### CODE REVIEW

File: src/_content/pages/errorpage.njk [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: 404 not found
+permalink: /error-page/index.html
```
Comment (alexcarpenter) [inline src/_content/pages/errorpage.njk]: Update permalink to `/404.html`

File: src/_content/pages/errorpage.njk [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+title: 404 not found
```
Comment (alexcarpenter) [inline src/_content/pages/errorpage.njk]: Update title to be just `404`

File: src/_content/pages/errorpage.njk [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+permalink: /error-page/index.html
+---
```
Comment (alexcarpenter) [inline src/_content/pages/errorpage.njk]: Lets make use of the markdown layout, similar to what you see here on the Discord Success page. https://raw.githubusercontent.com/alexcarpenter/frontend-foundations/master/src/_content/pages/discord-success.md

I think we could keep it really simple at the moment. Below is what you could add.

```yaml
---
layout: markdown
title: 404
description: Page not found
---
```

**Review by alexcarpenter — CHANGES_REQUESTED** [review]
Thanks for taking a stab at this. I have a few changes you can make before we get this merged in.

Lets also update the file name to be 404.md

**Comment by shoutoutloud** [comment]
So if I am not wrong, you want me to add layout, title and description to the commited   page and rename it to 404.md. Am I right?

**Comment by alexcarpenter** [comment]
@shoutoutloud Correct, rename the file you created to be `404.md` and all the needs to be included is

```yaml
---
layout: markdown
title: 404
description: Page not found
---
```

**Comment by shoutoutloud** [comment]
Sure, will do. Thanks.

**Comment by shoutoutloud** [comment]
Should I remove everything else from the file apart from layout , title and description?

**Comment by alexcarpenter** [comment]
@shoutoutloud Yeah, for now that should work. I might add some more helpful content within the page, but I am not sure what that should be at the moment.

**Comment by shoutoutloud** [comment]
Done.Please see.

**Comment by alexcarpenter** [comment]
@shoutoutloud Great, lets delete the unused error page you created and then we can get this merged.

**Comment by shoutoutloud** [comment]
How will I delete the page?

**Comment by alexcarpenter** [comment]
Delete the file highlighted here 
f3a7c01fcf286b76cd6b00f494e39957 _image 202018-07-02 20at 2010 46 21 20am

**Review by alexcarpenter — APPROVED** [review]

**Comment by shoutoutloud** [comment]
Thank you for your time and help.

**Pull request merged**